src/components/ModelDialog/components/ModalContent/ModalContent.tsx

```diff
@@ -0,0 +1,40 @@
+import type { ReactNode } from 'react'
+
+import { StyledDescription, StyledFormFooter, StyledSection } from './styled'
+
+export type Props = {
+  onClose: () => void
+  onConfirm?: () => void
+  description?: string
+  isConfirmation?: boolean
+  children?: ReactNode
+}
+
+export const ModalContent = ({
+  description,
+  isConfirmation,
+  onClose,
+  onConfirm,
+  children,
+}: Props) => {
+  const descriptionText = description
+    ? description
+    : 'Er is geen back-up beschikbaar.'
+  return (
+    <StyledSection data-testid="modal-content">
+      {children}
+      {isConfirmation && (
+        <>
+          <StyledDescription>{descriptionText}</StyledDescription>
```

Comment:
I would pass the description in the children props

Reply:
Removed description and am just showing children or defaultvalue if no children

Reply:
Done, sort of, removed it all together